### PR TITLE
🌐 Add French translation for `docs/fr/docs/advanced/additional-status-code.md`

### PR DESCRIPTION
**Pull request by axel584**
Here is the PR to translate the `advanced/additional-status-code.md`
See the french translation tracking issue https://github.com/tiangolo/fastapi/issues/1972.

Thanks for the reviews

### CODE REVIEW

**Comment by rjNemo** [comment]
Hi @axel584, you also need to update the docs/fr/mkdocs.yml file to include the newly translated page

File: docs/fr/docs/advanced/additional-status-codes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Codes de statut supplémentaires
+
+Par défaut, **FastAPI** renverra les réponses à l'aide d'une structure de données `JSONResponse`, en plaçant la réponse de votre  *chemin d'accès* à l'intérieur de cette `JSONResponse`.
+
+Il utilisera le code d'état par défaut ou celui que vous avez défini dans votre *chemin d'accès*.
```
Comment (rjNemo) [inline docs/fr/docs/advanced/additional-status-codes.md:5]: I would use _statut_ instead of _état_

**Review by rjNemo — CHANGES_REQUESTED** [review]

File: docs/fr/docs/advanced/additional-status-codes.md (hunk 1 above)
Comment (axel584) [reply docs/fr/docs/advanced/additional-status-codes.md:5]: @rjNemo : Wikipedia parle de "codes HTTP" (sans parler de _statut_ ou _d'état_) : [Liste des codes HTTP](https://fr.wikipedia.org/wiki/Liste_des_codes_HTTP)

Qu'en dis-tu ?

Comment (rjNemo) [reply docs/fr/docs/advanced/additional-status-codes.md:5]: Encore mieux!

**Comment by axel584** [comment]
Le breaking change in the unit test will be resolve by this PR : https://github.com/tiangolo/fastapi/pull/5456

**Review by emmrichard — APPROVED** [review]

File: docs/fr/docs/advanced/additional-responses.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+```JSON hl_lines="3-12"
+{
+    "responses": {
+        "404": {
+            "description": "Réponse supplémentaire",
+            "content": {
+                "application/json": {
+                    "schema": {
+                        "$ref": "#/composants/schémas/Message"
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/additional-responses.md:55]: Why not about the description but please do not translate the ref, we need to keep consistent with the example (`{!../../../docs_src/additional_responses/tutorial001.py!}`, [here ](https://fastapi.tiangolo.com/fr/advanced/additional-responses/#additional-response-with-model))
```suggestion
                        "$ref": "#/components/schemas/Message"
```

this suggestion applicable for the following refs as well

File: docs/fr/docs/advanced/additional-responses.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+                }
+            }
+        }
+    }
+}
+```
+
+Les schémas sont référencés à un autre endroit du modèle OpenAPI :
+
+```JSON hl_lines="4-16"
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/additional-responses.md:86]: Same here, we don't need to translate the full JSON. Some keys are also translated (eg. `Propriétés`) ⚠️

File: docs/fr/docs/advanced/additional-responses.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+Vous pouvez utiliser ce même paramètre `responses` pour ajouter différents types de médias pour la même réponse principale.
+
+Par exemple, vous pouvez ajouter un type de média supplémentaire `image/png`, en déclarant que votre *opération de chemin* peut renvoyer un objet JSON (avec le type de média `application/json`) ou une image PNG :
+
+```Python hl_lines="19-24 28"
+{!../../../docs_src/additional_responses/tutorial002.py!}
+```
+
+!!! Remarque
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/additional-responses.md:175]: Not sure the panel rendering will work translating it
Did you generate the documentation locally to see the result? 

Applicable to others `!!! note` and `!!! info`

File: docs/fr/docs/advanced/additional-responses.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+<img src="/img/tutorial/additional-responses/image01.png">
+
+## Combinez les réponses prédéfinies et les réponses personnalisées
+
+Vous voulez peut-être avoir des réponses prédéfinies qui s'appliquent à de nombreux *paramètre de chemin*, mais vous souhaitez les combiner avec des réponses personnalisées nécessaires à chaque *opération de chemin*.
+
+Dans ces cas, vous pouvez utiliser la technique Python "d'affection par décomposition" d'un `dict` avec `**dict_to_unpack` :
+
+``` Python
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/additional-responses.md:209]: This one is less impacting than the previous, but still I think we can avoid the translation in all the examples

File: docs/fr/docs/advanced/additional-responses.md [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+```Python hl_lines="13-17 26"
+{!../../../docs_src/additional_responses/tutorial004.py!}
+```
+
+## Plus d'informations sur les réponses OpenAPI
+
+Pour voir exactement ce que vous pouvez inclure dans les réponses, vous pouvez consulter ces sections dans la spécification OpenAPI :
+
+* <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responsesObject" class="external-link" target="_blank">Objet de réponses OpenAPI </a>, il inclut le `Response Object`.
+* <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responseObject" class="external-link" target="_blank">Objet de réponse OpenAPI </a>, vous pouvez inclure n'importe quoi directement dans chaque réponse à l'intérieur de votre paramètre `responses`. Y compris `description`, `headers`, `content` (à l'intérieur de cela, vous déclarez différents types de médias et schémas JSON) et `links`.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/additional-responses.md:240]: Wdyt ? The uppercase R let me think that we are referring to the object class name.
```suggestion
* <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responsesObject" class="external-link" target="_blank">Objet Responses de OpenAPI </a>, il inclut le `Response Object`.
* <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responseObject" class="external-link" target="_blank">Objet Response de OpenAPI </a>, vous pouvez inclure n'importe quoi directement dans chaque réponse à l'intérieur de votre paramètre `responses`. Y compris `description`, `headers`, `content` (à l'intérieur de cela, vous déclarez différents types de médias et schémas JSON) et `links`.
```

File: docs/fr/docs/advanced/additional-status-codes.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+!!! note "Détails techniques"
+    Vous pouvez également utiliser `from starlette.responses import JSONResponse`.
+
+    Pour plus de commodités, **FastAPI** fournit les objets `starlette.responses` sous forme d'un alias accessible par `fastapi.responses`. Mais la plupart des réponses disponibles proviennent directement de Starlette. Il en est de même avec l'objet `statut`.
+
+## Documents OpenAPI et API
+
+Si vous renvoyez directement des codes HTTP et des réponses supplémentaires, ils ne seront pas inclus dans le schéma OpenAPI (la documentation de l'API), car FastAPI n'a aucun moyen de savoir à l'avance ce que vous allez renvoyer.
+
+Mais vous pouvez documenter cela dans votre code, en utilisant : [Réponses supplémentaires dans OpenAPI](additional-responses.md){.internal-link target=_blank}.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/additional-status-codes.md:37]:
```suggestion
Mais vous pouvez documenter cela dans votre code, en utilisant : [Reponses supplémentaires](additional-responses.md){.internal-link target=_blank}.
```

**Review by JulianMaurin — COMMENTED** [review]
Small remarks (do not hesitate to challenge them) but this is nice 🙌
Thanks !

File: docs/fr/docs/advanced/additional-responses.md (hunk 4 above)
Comment (axel584) [reply docs/fr/docs/advanced/additional-responses.md:175]: yes, and it's look great : https://6346f37f0775e9008d2e2df2--fastapi.netlify.app/fr/advanced/additional-responses/

File: docs/fr/docs/advanced/additional-responses.md (hunk 3 above)
Comment (axel584) [reply docs/fr/docs/advanced/additional-responses.md:86]: I agree !

File: docs/fr/docs/advanced/additional-responses.md (hunk 5 above)
Comment (axel584) [reply docs/fr/docs/advanced/additional-responses.md:209]: yes, you're right, the mix between French and English is weird. I left the code in English and specified the term "unpacking" for the explanations.

File: docs/fr/docs/advanced/additional-status-codes.md (hunk 7 above)
Comment (axel584) [reply docs/fr/docs/advanced/additional-status-codes.md:37]: I don't understand why you want to remove the "é" in "Réponses"... and I prefer use the whole title which specify "dans OpenAPI"...

**Review by emmrichard — APPROVED** [review]

**Review by rjNemo — APPROVED** [review]

**Comment by axel584** [comment]
@JulianMaurin / @rjNemo : Does anyone know why my MR did not receive the "approved-2" label?

**Comment by tiangolo** [comment]
Thank you @axel584! :cake: 

And thanks for the reviews @JulianMaurin, @emmrichard, @rjNemo :coffee:

**Pull request merged**